src/query/api/v1/handler/prometheus/native/common.go

```diff
@@ -83,21 +72,21 @@ func parseParams(
 	params.Now = time.Now()
 	if v := r.FormValue(timeParam); v != "" {
 		var err error
-		params.Now, err = ParseTime(r, timeParam, params.Now)
+		params.Now, err = prometheus.ParseTime(r, timeParam, params.Now)
```

Comment:
Moved this method to `query/api/v1/handler/prometheus/common.go` to help resolve an import cycle. This is arguably a better place anyway.

Reply:
True yeah.